telemetry/telemetry.go

```diff
@@ -209,3 +210,19 @@ func getflag(name string) string {
 	})
 	return val
 }
+
+func validateSHA512(file string) {
```

Comment:
This function performs two tasks: it computes the checksum of a given file using the SHA-512 algorithm and it logs a message to the system logger with the file name and the checksum value. Therefore, the function name validateSHA512 is misleading, as it implies that the function only validate checksum.

Reply:
Changed